src/sdk/tracing/index.ts

```diff
@@ -173,6 +195,18 @@ const manuallyInitInstrumentations = (
   }
 };
 
+// Next.js instrumentation is very verbose and can result in
+// a lot of noise in the traces. By default, Laminar
+// will ignore the Next.js spans (looking at the attributes like `next.span_name`)
+// and set the topmost non-Next span as the root span in the trace.
+// Here's the set of possible attributes as of Next.js 15.1
+// See also: https://github.com/vercel/next.js/blob/790efc5941e41c32bb50cd915121209040ea432c/packages/next/src/server/lib/trace/tracer.ts#L297
+const isNextSpan = (span: ReadableSpan) => {
```

Comment:
let's switch to NextJs (here and everywhere) for consistency, otherwise I read it as actual next span.